NEWS

```diff
@@ -11,6 +11,11 @@
 ### Features:
 ### Bugfixes:
 
+## 1.14.0 RC2 (February 2, 2023)
```

Comment:
RC2 should replace RC1 in the NEWS, and also add this to the full list of bugfixes

Reply:
but we made separate sections with just relevant updates for v.12-rc* and v13-rc* series.
E.g. [here](https://github.com/openucx/ucx/releases/tag/v1.12.0-rc2)